DEVELOPMENT.md

```diff
@@ -107,7 +105,7 @@ curl http://localhost:5984 # should fail
 To be able to use Fauxton with authenticated users:
 
 ```shell
-curl -X PUT "http://myadminuser:myadminpass@localhost:5984/_node/$COUCH_NODE_NAME/_config/httpd/WWW-Authenticate" \
+curl -X PUT "http://myadminuser:myadminpass@localhost:5984/_node/_local/_config/httpd/WWW-Authenticate" \
```

Comment:
This is fine for the scope of this issue, but ultimately all of this should be scripted as part of the image setup, right?

Reply:
yeap, correct.